core/src/main/scala/blueprint/Render.scala

```diff
@@ -2,49 +2,147 @@ package nelson
 package blueprint
 
 import nelson.Datacenter.StackName
+import nelson.docker.Docker
 import nelson.Manifest._
 import nelson.blueprint.EnvValue._
 import nelson.docker.Docker.Image
 
+import cats.Id
+import cats.instances.string._
+import cats.syntax.eq._
+
 object Render {
-  def makeEnv(img: Image, dc: Datacenter, ns: NamespaceName, unit: UnitDef, v: Version, p: Plan, hash: String): Map[String, EnvValue] = {
-    import Render.keys._
 
-    val sn = StackName(unit.name, v, hash)
+  type Partial[F[_]] = F[Map[String, EnvValue]]
+
+  def base(img: Image, dc: Datacenter, ns: NamespaceName, unit: UnitDef, plan: Plan, sn: StackName): Partial[Id] =
+    makeEnv(img, dc, ns, unit, plan, sn) ++
+    makeJob(unit, plan) ++
+    makeResources(unit, plan)
+
+  def canopus(img: Image, dc: Datacenter, ns: NamespaceName, unit: UnitDef, plan: Plan, sn: StackName): Partial[Id] =
+    base(img, dc, ns, unit, plan, sn) ++
+    makeLivenessProbe(plan.environment)
+
+  def magnetar(img: Image, dc: Datacenter, ns: NamespaceName, unit: UnitDef, plan: Plan, sn: StackName, dk: Infrastructure.Docker): Partial[Id] =
+    base(img, dc, ns, unit, plan, sn) ++
+    makeDocker(dk) ++
+    makeNomadHealthChecks(plan.environment) ++
+    makeMetadata(unit) ++
+    makeVault(List(vault.policies.policyName(sn, ns)), "restart", "")
+
+  def pulsar(img: Image, dc: Datacenter, ns: NamespaceName, unit: UnitDef, plan: Plan, sn: StackName): Partial[Id] =
+    canopus(img, dc, ns, unit, plan, sn)
```

Comment:
This doesn't look right. See the workflow matrix: https://getnelson.io/getting-started/blueprints.html#workflows - where's the Vault support?